<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>681</width>
    <height>524</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="area1" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>671</width>
     <height>531</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: qlineargradient(spread:pad, 
                                x1:0, y1:0, x2:0, y2:1, 
                                stop:0 rgba(6, 6, 6, 255), 
                                stop:0.41 rgba(56, 50, 52, 255), 
                                stop:1 rgba(34, 33, 33, 255));;
</string>
   </property>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>10</y>
      <width>341</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Azonix</family>
      <pointsize>22</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background:transparent;</string>
    </property>
    <property name="text">
     <string>filter Match :</string>
    </property>
   </widget>
   <widget class="QPushButton" name="applyFilterButton">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>420</y>
      <width>131</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
background-color:qlineargradient(spread:pad, 
                            x1:0, y1:0, x2:1, y2:1, 
                            stop:0 rgba(255, 0, 0, 255), 
                            stop:0.41 rgba(195, 22, 58, 255), 
                            stop:1 rgba(255, 117, 0, 255));;  ;  
border-radius:10px;
color:white;
}
QPushButton:hover {
    background-color:qlineargradient(x1: 0.25, y1: 0, x2: 0.75, y2: 1, 
                            stop: 0 rgba(0, 255, 254, 255),     
                            stop: 0.26 rgba(38, 170, 196, 255), 
                            stop: 0.67 rgba(89, 85, 131, 255),
                            stop: 1 rgba(0, 238, 255, 255));  
}</string>
    </property>
    <property name="text">
     <string>applyFilter</string>
    </property>
   </widget>
   <widget class="QPushButton" name="cancel">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>420</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
background-color:qlineargradient(spread:pad, 
                            x1:0, y1:0, x2:1, y2:1, 
                            stop:0 rgba(255, 0, 0, 255), 
                            stop:0.41 rgba(195, 22, 58, 255), 
                            stop:1 rgba(255, 117, 0, 255));;  ;  
border-radius:10px;
color:white;
}
QPushButton:hover {
    background-color:qlineargradient(x1: 0.25, y1: 0, x2: 0.75, y2: 1, 
                            stop: 0 rgba(0, 255, 254, 255),     
                            stop: 0.26 rgba(38, 170, 196, 255), 
                            stop: 0.67 rgba(89, 85, 131, 255),
                            stop: 1 rgba(0, 238, 255, 255));  
}</string>
    </property>
    <property name="text">
     <string>cancelFilter</string>
    </property>
   </widget>
   <widget class="QLabel" name="matchIDLabel">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>10</y>
      <width>321</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Azonix</family>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background:transparent;
color:yellow;
</string>
    </property>
    <property name="text">
     <string>label</string>
    </property>
   </widget>
   <widget class="QComboBox" name="stadiumComboBox">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>140</y>
      <width>191</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QComboBox {
    background-color: white;
    padding: 5px;
    color: black;
    font-size: 16px;
    border: 2px solid transparent; /* Default border */
}

/* Drop-down arrow area */
QComboBox::drop-down {
    border: none;
    width: 30px;
   ; /* Keep background for drop-down */
    border-top-right-radius: 8px;
    border-bottom-right-radius: 8px;
    image: url(:/textures/textures/downArrowIcon.png);
}

/* Hover effect */
QComboBox:hover {
    border: 3px dashed qlineargradient(spread:pad, 
                            x1:0, y1:0, x2:1, y2:1, 
                            stop:0 rgba(255, 0, 0, 255), 
                            stop:0.41 rgba(195, 22, 58, 255), 
                            stop:1 rgba(255, 117, 0, 255));
}

/* Focus effect */
QComboBox:focus {
    border: 2px solid qlineargradient(spread:pad, 
                            x1:0, y1:0, x2:1, y2:1, 
                            stop:0 rgba(255, 0, 0, 255), 
                            stop:0.41 rgba(195, 22, 58, 255), 
                            stop:1 rgba(255, 117, 0, 255));
}

/* Style for the dropdown list */
QComboBox QAbstractItemView {
    background-color: white;
    border: 2px solid #C51D34;
    selection-background-color: #ff0004;
    selection-color: white;
    border-radius: 8px;
    color: black;
}
</string>
    </property>
    <property name="placeholderText">
     <string>Stadium</string>
    </property>
   </widget>
   <widget class="QDateEdit" name="startDateEdit">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>230</y>
      <width>191</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QDateEdit {
    background-color: white;
    padding: 5px;
    color: black;
    font-size: 16px;
    border: 2px solid transparent; /* Default border */
}

/* Drop-down arrow area */

/* Hover effect */
QDateEdit:hover {
    border: 3px dashed qlineargradient(spread:pad, 
                            x1:0, y1:0, x2:1, y2:1, 
                            stop:0 rgba(255, 0, 0, 255), 
                            stop:0.41 rgba(195, 22, 58, 255), 
                            stop:1 rgba(255, 117, 0, 255));
}


</string>
    </property>
   </widget>
   <widget class="QDateEdit" name="endDateEdit">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>320</y>
      <width>191</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QDateEdit {
    background-color: white;
    padding: 5px;
    color: black;
    font-size: 16px;
    border: 2px solid transparent; /* Default border */
}

/* Drop-down arrow area */

/* Hover effect */
QDateEdit:hover {
    border: 3px dashed qlineargradient(spread:pad, 
                            x1:0, y1:0, x2:1, y2:1, 
                            stop:0 rgba(255, 0, 0, 255), 
                            stop:0.41 rgba(195, 22, 58, 255), 
                            stop:1 rgba(255, 117, 0, 255));
}


</string>
    </property>
   </widget>
   <widget class="QWidget" name="resultComboBox" native="true">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>170</y>
      <width>191</width>
      <height>131</height>
     </rect>
    </property>
    <widget class="QRadioButton" name="wonRadioButton">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>112</width>
       <height>26</height>
      </rect>
     </property>
     <property name="text">
      <string>win</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="lostRadioButton">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>112</width>
       <height>26</height>
      </rect>
     </property>
     <property name="text">
      <string>lose</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="drawRadioButton">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>90</y>
       <width>112</width>
       <height>26</height>
      </rect>
     </property>
     <property name="text">
      <string>draw</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
